management/src/test/java/org/ehcache/management/providers/statistics/StatsUtil.java

```diff
@@ -1,71 +0,0 @@
-/*
```

Comment:
There are options, the ones passing through my head currently look bad. But clearly leaking test code in production code is not an option.